packages/components/psammead-brand/src/index.jsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import styled, { css } from 'styled-components';
-import { string } from 'prop-types';
+import { string, number, node, shape } from 'prop-types';
 import { C_POSTBOX, C_WHITE } from '@bbc/psammead-styles/colours';
+import { news } from '@bbc/psammead-assets/svgs';
```

Comment:
I dont think we should have a default here, as it will bloat the bundle of other services. Its fine to make it a required prop :)